src/js/components/CheckBox/doc.js

```diff
@@ -20,6 +20,9 @@ export const doc = CheckBox => {
     checked: PropTypes.bool
       .description('Same as React <input checked={} />')
       .defaultValue(false),
+    defaultChecked: PropTypes.bool
+      .description('Same as React <input defaultChecked={} />')
+      .defaultValue(false),
```

Comment:
I thought our policy was not to document any React's given prop unless we modify them internally

Reply:
Good to know that policy/perspective. Would like to know the outcome of this discussion.

Reply:
Following the thread should do the trick.

Reply:
Well, I added it in because we are actually doing something with it, a bit like `defaultValue` for Select. However, I agree from an API point of view, the caller isn't being given any new information or behavior that they need to care about. I will remove it.